src/bindings/c/docs/OpenVINO_Runtime_C_API_User_Guide.md

```diff
@@ -0,0 +1,62 @@
+# User Guide of OpenVINO C API
+
+> **NOTE**: This guide provides a simplified instroduction about OpenVINO C API.
+C API provides simplified interface for OpenVINO functionality that allows to:
+
+- handle the input network models & input/output tensors.
+- load and configure OpenVINO plugins based on device name.
+- perform inference in synchronous and asynchronous modes with arbitrary number of infer requests (the number of infer requests may be limited by target device capabilities).
+
+## Supported OSes
+
+Currently the OpenVINO C API is supported on Ubuntu* 20.04, Microsoft Windows* 10 and CentOS* 7.3 OSes.
+Supported Python* versions:
+
+- On Ubuntu 20.04: 2.7, 3.5, 3.6
+- On Windows 10: 3.5, 3.6
+- On CentOS 7.3: 3.4, 3.5, 3.6
+
+## Instances Mapping Relationships
+| C Structure    | C++ Class  | Description |
+|:---     |:--- |:---
+| `ov_core_t` | `ov::Core` | An structure represents the 'ov::Core' pointer
+| `ov_compiled_model_t` | `ov::CompiledModel` | An structure represents the 'ov::CompiledModel' pointer
+| `ov_dimension_t` | `ov::Dimension` | An structure represents the 'ov::Dimension' object
+| `ov_infer_request_t` | `ov::InferRequest` | An structure represents the 'ov::InferRequest' pointer
+| `ov_layout_t` | `ov::Layout` | An structure represents the 'ov::Layout' object
+| `ov_model_t` | `ov::Model` | An structure represents the 'ov::Model' pointer
+| `ov_output_port_t`/`ov_output_const_port_t` | `ov::Output\<ov::Node\>`/`ov::Output\<const ov::Node\>` | Structure represents the `ov::Output\<ov::Node\>`/`ov::Output\<const ov::Node\>` pointer
+| `ov_partial_shape_t` | `ov::Dimension` | An structure represents the vector of `ov::Dimension`
+| `ov_preprocess_prepostprocessor_t` | `ov::preprocess::PrePostProcessor` | An structure represents the `ov::preprocess::PrePostProcessor` pointer
+| `ov_property_key_*` | `ov::Any` | An properties provided by OpenVINO
+| `ov_rank_t` | `ov::Dimension` | The size of `ov::Dimension`
+| `ov_shape_t` | `ov::shape` | An structure represents the `ov::shape` object
+| `ov_tensor_t` | `ov::Tensor` | An structure represents the `ov::Tensor` pointer
+
+
+## Provided APIs
```

Comment:
Why do we need it? It should be a part of API References which is generated by doxygen.